cfgov/v1/util/ref.py

```diff
@@ -77,8 +77,8 @@
         ('stipulation-and-consent-order-2', 'Stipulation and consent order'),
     )),
     ('Final Rule', (
```

Comment:
Should this one get updated, too? The top-level items in this `categories` object are wildly inconsistent.

Reply:
Actually, regardless of whether they're used for display purposes, it'd need to be in sync with the other ones given that it's used as a look-up table, so making the change here as well. Thanks for the catch.